src/components/MobileNavbar/index.tsx

```diff
@@ -1,73 +1,87 @@
 import Button from '../common/Button'
 import classNames from 'classnames'
+import Modal from '../../components/common/Modal'
+import ContactSalesForm from '../../components/ContactSalesForm'
+import styles from './MobileNavbar.module.scss'
 
 export default function MobileNavbar() {
+  const [isContactSalesModalOpen, setIsContactSalesModalOpen] = useState(false)
+
+  useEffect(() => {
+    document.body.style.overflow = isContactSalesModalOpen ? 'hidden' : 'unset'
+  }, [isContactSalesModalOpen])
+
   return (
-    <div className={styles.nav}>
-      <Container size='large' className={styles.container}>
-        <div className={classNames(styles.links, styles.top)}>
-          <Button href='mailto:sales@fingerprintjs.com' variant='outline'>
-            Contact Sales
-          </Button>
-          <Button href='https://dashboard.fingerprintjs.com/signup'>Free Trial</Button>
-        </div>
-        <div className={classNames(styles.links, styles.main)}>
-          <Container size='large' className={styles.container}>
-            {' '}
-            <Link to='/demo' className={styles.link}>
-              Technical Demo
-            </Link>
-            <a
-              href='https://github.com/fingerprintjs/fingerprintjs'
-              target='_blank'
-              rel='noreferrer'
-              className={styles.link}
-            >
-              Open Source
-            </a>
-            <Link to='/pricing' className={styles.link}>
-              Pricing
-            </Link>
-            <a href='https://dev.fingerprintjs.com' className={styles.link}>
-              Documentation
-            </a>
-            <a href='mailto:support@fingerprintjs.com' className={styles.link}>
-              Support
-            </a>
-            <a href='https://dashboard.fingerprintjs.com/login' className={styles.link}>
-              Log In
-            </a>
-            <a href='https://dashboard.fingerprintjs.com/signup' className={styles.link}>
-              Sign Up
-            </a>
-          </Container>
-        </div>
-        <div className={styles.contact}>
-          <GithubButton />
-          <div className={styles.social}>
-            <small>Find us on social</small>
-            <ul className={styles.links}>
-              <li className={styles.link}>
-                <a href='https://www.linkedin.com/company/fingerprintjs/' target='_blank' rel='noreferrer'>
-                  <LinkedInSvg />
-                </a>
-              </li>
-              <li className={styles.link}>
-                <a href='https://twitter.com/FingerprintJs' target='_blank' rel='noreferrer'>
-                  <TwitterSvg />
-                </a>
-              </li>
-            </ul>
+    <>
+      <div className={styles.nav}>
+        <Container size='large' className={styles.container}>
+          <div className={classNames(styles.links, styles.top)}>
+            <Button onClick={() => setIsContactSalesModalOpen(true)} variant='outline'>
```

Comment:
Maybe let's create Contact Sales custom button with Modal logic inside? To reduce copy paste logic

Reply:
Also looks like there is extra whitespace before Button tag.
We need to set up eslint and prettier extensions for you

Reply:
Weird, they are setup, I'll look into that.

![image](https://user-images.githubusercontent.com/10968161/99835591-9656b480-2b43-11eb-931b-9ad6a10ea079.png)

Reply:
Just double-checked, they are active and formatting on save. I'll keep an eye out for this and see if it happens again.

Reply:
They weren't formatting on save. Fixed.